jetty-client/src/test/java/org/eclipse/jetty/client/HttpClientTLSTest.java

```diff
@@ -1003,4 +1006,49 @@ public void testForcedNonDomainSNI() throws Exception
             assertEquals(HttpStatus.OK_200, response3.getStatus());
         }
     }
+
+    @Test
+    public void testBytesInBytesOut() throws Exception
+    {
+        // Two connections will be closed: SslConnection and HttpConnection.
+        // Two on the server, two on the client.
```

Comment:
@gregw if you look at the test here it seems to indicate that both `SslConnection` and `HttpConnection` are recorded by `ConnectionStatistics`.

`ConnectionStatistics` works as a `Connection.Listener` so it would apply to `SslConnection` as well.

Reply:
Hmmm is this different on the Server-side?  As I don't see how a Connection.Listener would see any connections wrapped by SslConnection?

Reply:
The `Connection.Listener` is added as a bean to all connectors on the server, and any `Connection.Listener` beans on the Connector are added to each new connection that is created. So the listener will automatically be added to the `SslConnection`.

Reply:
Yeah, so the listener is on SslConnection. Is it also inherited by the Wrapped HttpConnection? If so, then i agree the double counting is wrong.